Assets/Scripts/Tools/GraphArrangerTool.cs

```diff
@@ -47,6 +47,6 @@ public void OnSelect()
 
     private Color GetPanelColor() => _graphArranger.ArrangeEnabled ? Color.green : Color.red;
 
-    private string GetArrangerMode() => EnumUtils<GraphArrangerMode>.GetName(_graphArranger.ArrangeMode).Substring(1);
+    private string GetArrangerModeLabel() => EnumUtils<GraphArrangerMode>.GetName(_graphArranger.ArrangeMode).Substring(1);
```

Comment:
why .Substring(1) ?
I think it would be clearer if this call to substring would be inside another method (with meaningful name) in EnumUtils

Reply:
fixed with enum extension method